seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -660,13 +660,13 @@
         "<code>    }</code>",
         "<code>}</code>",
         "We can remove the function keyword and colon (:) altogether - and get this:",
-        " const Container extends Component {",
-        "     render() {",
-        "       return {",
-        "         `<div>Container</div>`",
-        "       }",
-        "     }",
-        " }",
+        "<code> const Container extends Component { </code>",
```

Comment:
@erictleung So, I tried out the `<blockquote>` on the first code example. Please review this, I can move on to the next example if this one looks all right - 

![screenshot 2017-02-06 18 58 43](https://cloud.githubusercontent.com/assets/11348778/22671883/a66c2b4e-ec9e-11e6-9ee7-72980c86c53c.png)